### ISSUE TITLE
sql: add show transactions

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/cockroach#51885]
We currently have `SHOW SESSIONS` and `SHOW QUERIES` but we do not have a corresponding `SHOW TRANSACTIONS`. We should add this capability. For now, we have `SELECT * FROM crdb_internal.cluster_transactions;`  available but we can improve upon this to make it easier for developers to pull this information from the SQL shell. 

```SQL
root@127.0.0.1:65336/movr> SHOW SESSIONS;
  node_id |            session_id            | user_name | client_address  | application_name |    active_queries     | last_active_query |          session_start           |        oldest_query_start
----------+----------------------------------+-----------+-----------------+------------------+-----------------------+-------------------+----------------------------------+-----------------------------------
        1 | 1624c3d81675d7200000000000000001 | root      | 127.0.0.1:65365 | $ cockroach demo | SHOW CLUSTER SESSIONS | SHOW database     | 2020-07-24 18:31:40.853051+00:00 | 2020-07-24 18:40:01.013231+00:00
(1 row)

Time: 3.242ms
```
```SQL
root@127.0.0.1:65336/movr> SHOW QUERIES;
              query_id             | node_id |            session_id            | user_name |              start               |        query         | client_address  | application_name | distributed |   phase
-----------------------------------+---------+----------------------------------+-----------+----------------------------------+----------------------+-----------------+------------------+-------------+------------
  1624c453a57948c80000000000000001 |       1 | 1624c3d81675d7200000000000000001 | root      | 2020-07-24 18:40:31.533461+00:00 | SHOW CLUSTER QUERIES | 127.0.0.1:65365 | $ cockroach demo |    false    | executing
(1 row)

Time: 7.09ms
```
```SQL
root@127.0.0.1:65336/movr> SELECT * FROM crdb_internal.cluster_transactions;
                   id                  | node_id |            session_id            |              start               |                                                                                                  txn_string                                                                                                   | application_name
---------------------------------------+---------+----------------------------------+----------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------
  a7d238f3-01cd-44e4-8500-e6f8dc0b05b6 |       1 | 1624c3d81675d7200000000000000001 | 2020-07-24 18:40:50.198311+00:00 | "sql txn" meta={id=a7d238f3 key=/Min pri=0.00153114 epo=0 ts=1595616050.198302000,0 min=1595616050.198302000,0 seq=0} lock=false stat=PENDING rts=1595616050.198302000,0 wto=false max=1595616050.698302000,0 | $ cockroach demo
(1 row)

Time: 2.783ms
```
This also relates to the work for adding a transactions page into the WebUI (https://github.com/cockroachdb/cockroach/issues/46429). 

The SQL statement should at a minimum show:

- Session ID
- Application ID
- SQL statements that comprise the transaction
- Total time taken as of the running of this statement
- Number of retries
- Last retry encountered

**Comment by xuhui-lu** [comment]
hey Any, I am interested in this issue. May I take it?

**Comment by awoods187** [comment]
@xuhui-lu thank you for your interest. We already have a PR in progress https://github.com/cockroachdb/cockroach/pull/51902. I recommend you take a look at an issue marked `Easy` or `Good First Issue`

**Comment by jordanlewis** [comment]
Closed in #51902
